packages/ui/src/theme/overwrites/components/textarea.ts

```diff
@@ -0,0 +1,37 @@
+export const Textarea = {
+  baseStyle: {
+    width: '12.5rem',
+    height: '5rem',
+  },
+  variants: {
+    outline: {
+      borderRadius: '0.375rem',
```

Comment:
It would be great to provide all sizes non-related to fonts in pixels. This way we don't to recalculate it every time we want to check if the component match mockups